crates/typst/src/layout/flow.rs

```diff
@@ -1082,6 +1106,13 @@ impl<'a, 'e> FlowLayouter<'a, 'e> {
             && !self.items.is_empty()
             && self.items.iter().all(FlowItem::is_out_of_flow)
         {
+            let mut output = Frame::soft(self.initial);
+
+            if self.should_reset_page_scoped_line_numbers() {
+                // Reset line numbers at the first column if requested.
+                output.push_frame(Point::zero(), self.layout_line_number_reset()?);
+            }
```

Comment:
Then we can call that here as well.

Reply:
You mean the line number logic would apply here too, but with an empty vector of lines?

Reply:
okay i think that works, although I haven't added an explicit test for it yet. Resolving for now.

Reply:
Added a test for this now.